Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -153,30 +153,51 @@ class WalletConnectManager {
         }
     }
     
+    // Approves a single chain where selected safe resides.
+    //
+    // Requires:
+    //   - proposal with the chain that is same as the currently selected safe
+    // Guarantees:
+    //   - connection approved
+    //   - if no such chain found in proposal, connection will fail
     func approveSession(proposal: Session.Proposal) {
         Task {
             guard let safe = try? Safe.getSelected() else { return }
-            guard isValidProposal(proposal: proposal, safe: safe) else {
+                        
+            // Step 1: find a compatible namespace with safe's chain
+            var resultNamespaces = [String: SessionNamespace]()
+
+            let EVM_COMPATIBLE_NETWORK = "eip155"
```

Comment:
Why not reuse the static constant EVM_COMPATIBLE_NETWORK created above?

Reply:
oh I haven't removed the static variable, I see. I wanted to put the constant / var declaration closer to its usage